### DISCUSSION TITLE
Limitations about pipx-managed pipx?

### DISCUSSION THREAD

**Discussion by huxuan** [pypa/pipx#1341] [Q&A]
Hi pipx Community,

There is a warning about not recommended to install `pipx` via `pipx` in the documentation, but from my recent experience, `pipx` works quite well. So what is the exact limitation for pipx-managed pipx? I tried to search on the GitHub and Internet but did not find anything quite useful. The [`pipx-in-pix`] also seems to be outdated since there is no update for several years.

When working on Ubuntu 22.04, I firstly install pipx with `sudo apt install pipx` and then use `pipx install pipx` to override it since the package provided by Ubuntu is only version 1.0.0. After that, I also find running `sudo apt remove pipx` will still have `pipx` works for me and it removes a lot of unwanted system-level packages.

Moreover, when I am working on `pipx` project, I find `pipx install -e . --suffix .local` is quite convenient for quick test with command `pipx.local`. Actually this is not limited to `pipx` itself and is applied to all cli development.

**Comment by ghost** [comment]
A similar question [How to upgrade pipx on Ubuntu #1380](https://github.com/pypa/pipx/discussions/1380) and would like to know any possible drawbacks.

**Comment by Gitznik** [comment]
I thought I responded to this somewhere but I can't find it. IMO if we can't find any good reason anymore to have that warning, we should remove it and document the option to install pipx that way.

**Comment by zinc75** [comment]
Can now be easily solved using https://github.com/pypa/pipx/issues/1481#issuecomment-2583365034 and https://github.com/zinc75/pipx-1.6.0-backport-ubuntu-2404-lts , at least for 24.04 . 

I've put the recipe for backporting newer versions of Ubuntu packages for pipx , which may be applicable also for 22.04 . 

Best,
